Tests/Find-ImageName.Tests.ps1

```diff
@@ -4,33 +4,31 @@ Import-Module -Global -Force $PSScriptRoot/MockReg.psm1
 
 Describe 'Parse context from git repository' {
 
+    $tempFolder = New-RandomFolder
+    $script:moduleName = (Get-Item $PSScriptRoot\..\*.psd1)[0].BaseName
+
```

Comment:
I think this part should be removed. It is moved into the below removed `BeforeAll` so that there is something valid to do in `AfterAll` section.